src/devices/MAX31856/MAX31856.cs

```diff
@@ -0,0 +1,188 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
also please remove  ConsoleApp1 file from the PR

Reply:
I cant find where this file is thats weird

Reply:
I figured it out it has to do with the files name changing to Max31856 vice MAX31856 changed all files to reflect Max31856

Reply:
One last thing the main src folder of MAX31856 is wrong I dont know how to change the folder name in src to Max31856 and push it

Reply:
do two commits:
- rename to MAX31856-2
- commit
- rename to whatever